shared/src/balancer.rs

```diff
@@ -0,0 +1,2 @@
+pub mod event_handler;
+pub mod pool_fetching;
```

Comment:
I bet you're happy I went this this instead of `mod.rs` I am considerate of your concerns with having too many mod.rs files. Although I am not sure why you would ever be searching for a mod.rs file.